packages/pymedphys_analysis/src/pymedphys_analysis/tpscompare/normalisation.py

```diff
@@ -0,0 +1,175 @@
+# Copyright (C) 2016 Cancer Care Associates
```

Comment:
Just wanted to give you a heads up here. "OLD CODE WARNING". Don't judge me too harshly. ... but also, feel free to not go easy.